integration_tests/book_gorm_redis/gorm_redis_int_nonuniquekey_test.go

```diff
@@ -217,6 +217,43 @@ func (s *GormRedisIntegrationNonUniqueKeyTestSuite) TestGetUpdateAndGetTwoAuthor
 	})
 }
 
+func (s *GormRedisIntegrationNonUniqueKeyTestSuite) TestGetPartialUpdateAndGetTwoAuthorsBooks() {
+	ctx := s.system.Ctx
+
+	Convey("Scenario: Retrieve books for 2 authors from cache with partial update", s.T(), func() {
+		Convey("Given an author with 2 books in the system, "+
+			"And an author without books, "+
+			"And I query for the books of the authors once", func() {
+			author1 := testdomain.CreateAuthor(s.system)
+			author2 := testdomain.CreateAuthor(s.system)
+			book1, _ := author1.CreateBook(ctx, EmptyStatus)
+			book2, _ := author1.CreateBook(ctx, CompletedStatus)
+
+			authorIds := []string{author1.AuthorId, author2.AuthorId}
+
+			testdomain.GetBooksForAuthors(s.system, authorIds)
+
+			Convey("When I update the 1st book for the 1st author and query for the books again", func() {
```

Comment:
minor - `When I do a partial update of...`

Reply:
done :)